bundles/org.openhab.binding.sbus/README.md

```diff
@@ -39,15 +38,20 @@ The Sbus Bridge has the following configuration parameters:
 
 ### Device Configuration
 
-All device types (RGBW Controller, Contact Sensor, Switch, Temperature, Motion Sensor, Lux Sensor) share the same configuration parameters:
+All device types share the same basic configuration parameters:
 
 | Name    | Type    | Description                                          | Default | Required | Advanced  |
 |:--------|:--------|:-----------------------------------------------------|:-------:|:--------:|:---------:|
 | subnetId| integer | Subnet ID the device is part of                      | N/A     | yes      | no        |
 | id      | integer | Device ID                                            | N/A     | yes      | no        |
 | refresh | integer | Refresh interval in seconds (0 = listen-only mode)   | 30      | no       | yes       |
 
-**Note:** Setting `refresh=0` enables listen-only mode for all device types. In this mode, handlers only process asynchronous broadcast messages without actively polling the devices. This is particularly useful for Motion Sensor and Lux Sensor devices that broadcast status updates (0x02CA) from 9-in-1 devices.
+**Device Type Specific Notes:**
+
+- **Traditional Contact Sensors (`contact-sensor`)**: Use ReadDryChannelsRequest protocol for simple contact monitoring
+- **Multi-Sensor Devices (`multi-sensor`)**: Use ReadNineInOneStatusRequest protocol and support motion, lux, and dry contact channels from a single physical device
+
+**Listen-Only Mode:** Setting `refresh=0` enables listen-only mode where handlers only process asynchronous broadcast messages (MotionSensorStatusReport) without actively polling. This is particularly useful for 9-in-1 sensor devices that broadcast status updates.
```

Comment:
To me this sounds like the 9-in1 sensor should have a own configuration with refresh set to 0 by default.